src/isilon_hadoop_tools/identities.py

```diff
@@ -13,6 +13,7 @@
     # Functions
     'cdh_identities',
     'hdp_identities',
+    'cdp_identities',
```

Comment:
Let's keep these alphabetized.
```suggestion
    'cdp_identities',
    'hdp_identities',
```